react/components/Modal/Modal.jsx

```diff
@@ -30,6 +42,9 @@ const Modal = ({
             overlayClassName="qpp-c-modal__overlay"
             bodyOpenClassName="qpp-c-modal__body-open"
             isOpen={isOpen}
+            style={{
+                content: getWidth(),
```

Comment:
just to confirm, this has been tested/verified with other places aside from HR where the modal is being used that this style is desired?

Reply:
@nmorasb Modal is only used in ExportModal, and as long as it's not passing in a width, it should be used as a regular modal (as shown in the first modal in the above screenshots). So the regular modal still works as before, for HR we'll need to pass in a width.

Reply:
got it, sounds good!